DeepSpeech.py

```diff
@@ -432,14 +445,16 @@ def train():
             FLAGS.augmentation_speed_up_std > 0 or
             FLAGS.augmentation_sparse_warp):
         do_cache_dataset = False
+    
+    drop_source_layers = ['2', '3', 'lstm', '5', '6'][-int(FLAGS.drop_source_layers):]
```

Comment:
So `--drop_source_layers 1` will remove layer `'2'`, and `--drop_source_layers 3` will remove `'2', '3' and 'lstm'`. That makes sense after reading the help message, but I think it's unobvious and it might trick people.

Could we either be more explicit in the help message, or not have `--drop_source_layers` be a magic number but directly a list of layers ? and thus `--drop_source_layers 2,3,lstm` for example ?

Reply:
You got it reversed, `--drop_source_layers 1` will drop layer '6' (note the negative index and the Python slicing syntax). So it'd say it's not obvious, even after reading help message :)

I'd vote for a number rather than a list of layers, as we don't want to expose variable names to users through the flags. How about `--number_of_layers_to_drop` and then document in the help message that it goes from the output layer to the input layer?